CMakeLists.txt

```diff
@@ -181,6 +181,10 @@ target_link_libraries(sof PRIVATE sof_options)
 target_link_libraries(sof PRIVATE sof_ld_scripts)
 target_link_libraries(sof PRIVATE sof_ld_flags)
 target_link_libraries(sof PRIVATE sof_static_libraries)
+# link codec library for codec_adapter to use
+if(CONFIG_COMP_CODEC_ADAPTER)
```

Comment:
I made a mistake here, should be **CONFIG_CADENCE_CODEC** of course. Will correct...